www/src/pages/404.astro

```diff
@@ -0,0 +1,60 @@
+---
+import LandingPage from "../layouts/landingPage.astro";
+import Button from "../components/landingPage/button.astro";
+---
+
+<LandingPage>
```

Comment:
Not sure where we pass in the flag to hide the themeToggler, but it should prob be hidden here (behavior is weird)

Reply:
Yeah, I'm trying to fix the `isLanding` things everywhere